src/modules/material-requests/services/material-requests.service.ts

```diff
@@ -204,17 +204,19 @@ export class MaterialRequestsService {
 		userProfileId: string,
 		materialRequestInfo: Pick<
 			App_Material_Requests_Set_Input,
-			'type' | 'reason' | 'organisation' | 'requester_capacity' | 'is_pending'
+			'type' | 'reason' | 'organisation' | 'requester_capacity' | 'is_pending' | 'updated_at'
 		>
 	): Promise<MaterialRequest> {
-		const { type, reason, organisation, requester_capacity, is_pending } = materialRequestInfo;
+		const { type, reason, organisation, requester_capacity, is_pending, updated_at } =
+			materialRequestInfo;
 
 		const updateMaterialRequest = {
 			type,
 			reason,
 			organisation,
 			requester_capacity,
 			is_pending,
+			updated_at,
```

Comment:
It seems strange to me that you would receive a value **updated_at** from the FE. In most use cases you retrieve all the changing data from FE and set a new value for **updated_at** in the BE.

REQ: remove option to retrieve **updated_at**

Reply:
The value is set in the controller. We can also update the 'updated_at ' in the service but that's not what is requested in the ticket.